### DISCUSSION TITLE
examples can't work

### DISCUSSION THREAD

**Discussion by algesthesiah** [vikejs/vike#1273] [Discussion] [answered]

**Comment by brillout** [comment]
Follow the instructions of the error

**Comment by phonzammi** [comment, marked as answer]
Hi @brillout  and @algesthesiah,

I found the problem,
it is inside onRenderHtml from vike-react package,
there is a condition checking : 
FYI @AurelienLourot 
```
const isSsrDisabled = !pageContext.Page; 
// ssr always true unless we explicitly set to false inside config right ?

const streamOrString = isSsrDisabled
        ? dangerouslySkipEscape(renderToString(htmlContent))
        : await renderToStream(htmlContent, { userAgent: pageContext.userAgent });
```
The thing is if we don't create our own `server/index.ts` file (server entry if i am not mistaken), it will just works.
But if we do (like in this example) then we have to return 

```
httpResponse.pipe(res); 
// res.status(statusCode).send(body) will cause an error
```

And in this example it is
```
 res.status(statusCode).send(body)
```

**Reply by brillout** [reply]
Indeed that was the error.

**Reply by lourot** [reply]
For future travelers: fixed by @phonzammi in #1274, thanks again @phonzammi!

**Reply by phonzammi** [reply]
Glad To Help. Thanks too, I've learned much too

**Reply by brillout** [reply]
💯
